### DISCUSSION TITLE
[HxAutosuggest] Dropdown Expands to Window Width When Position is Fixed

### DISCUSSION THREAD

**Discussion by TPIvan** [havit/Havit.Blazor#897] [Q&A] [answered]
When `data-bs-popper-config='{"strategy":"fixed"}'` is set on `HxAutosuggest`, the `dropdown` list expands to the full width of the window, as shown in the image below:
![image](https://github.com/user-attachments/assets/27a072b6-fcd8-45a3-b693-584873155f55)
Instead, it should look like this:
![image](https://github.com/user-attachments/assets/2e1b2d7c-ad3d-4f58-9bf7-9be5c29e057b)
The issue appears to be related to setting the dropdown's min-width to 100% (`--hx-autosuggest-dropdown-menu-width`).

A partial workaround is to set `--hx-autosuggest-dropdown-menu-width: initial;` (or another suitable value), which results in a slightly improved dropdown width:
![image](https://github.com/user-attachments/assets/48489f28-1a8c-454a-81b5-368f4c02fdd4)
However, the `dropdown` width still does not match the input.

I am unsure if this is an issue with the Havit.Blazor library or a problem of Bootstrap.

The problem is also present in `HxInputTags` and `HxMultiSelect`.

**Comment by crdo** [comment, marked as answer]
@TPIvan sadly, there is nothing we can do other than what you did. With position fixed, everything is positioned against the browser window and that applies for the size of the dropdown as well.
I wouldn't say it is either Havit.Blazor or Bootstrap fault.
I am waiting the Bootstrap team to migrate from Popper to its successor https://floating-ui.com/, but I am afraid that we are talking V6 -> long way to go.
